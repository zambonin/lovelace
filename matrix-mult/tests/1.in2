10 15 30
0 2 1
0 3 5
0 9 10
0 14 2
1 0 2
1 1 5
1 2 1
1 10 4
2 3 9
2 7 6
2 10 1
2 12 10
2 13 2
3 1 2
3 4 8
3 5 7
3 13 9
4 8 10
4 9 1
5 2 1
6 0 1
6 3 3
6 5 10
6 11 7
7 11 2
8 2 3
8 7 9
8 12 2
9 10 6
9 11 8
