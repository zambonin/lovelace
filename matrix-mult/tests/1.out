10 15 58
0 0 8
0 1 20
0 2 4
0 3 18
0 7 12
0 10 18
0 12 20
0 13 4
1 0 20
1 1 50
1 2 10
1 8 80
1 9 8
1 10 40
2 0 1
2 2 36
2 3 33
2 5 10
2 7 90
2 9 60
2 11 7
2 12 20
2 14 12
3 0 5
3 1 14
3 2 9
3 3 15
3 4 56
3 5 99
3 11 35
3 13 63
4 2 30
4 3 90
4 7 123
4 10 10
4 12 114
4 13 20
5 10 30
5 11 40
6 1 8
6 4 32
6 5 28
6 13 36
7 1 16
7 4 64
7 5 56
7 13 72
8 0 10
8 1 8
8 3 30
8 4 32
8 5 128
8 11 70
8 13 36
9 0 10
9 1 25
9 2 9
9 10 20
